doc/cla/individual/chowbryan

```diff
@@ -0,0 +1,13 @@
+I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
+
+United States, 2/3/2022
```

Comment:
could you use the `year-month-day` format? the american date format is unclear (3rd of Feb or 2nd of March)